operator/config/watcher/patches/watcher_service_patch.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/serviceInfo/name
+  value: lifecycle-manager-event-service
```

Comment:
same as what I mentioned in module-manager pr, the nameprefix is different for KCP deployment, we can't hard coded to lifecycle-manager-event-service

Reply:
ok then we need to do it on the base layer.